dts/arm/atmel/saml21.dtsi

```diff
@@ -7,29 +7,205 @@
 #include <atmel/saml2x.dtsi>
 
 / {
+
+	aliases {
+		watchdog0 = &wdog;
+	};
+
+	chosen {
+		zephyr,flash-controller = &nvmctrl;
+		zephyr,entropy = &trng;
+	};
+
 	soc {
-		usb0: usb@41000000 {
-			compatible = "atmel,sam0-usb";
-			status = "disabled";
-			reg = <0x41000000 0x1000>;
-			interrupts = <6 0>;
-			num-bidir-endpoints = <8>;
+
+		nvmctrl: nvmctrl@41004000  {
```

Comment:
nvmctrl@41004000 is the same for L21/22. There is a big error at Figure 8-1. SAM L22 Product Mapping datasheet on the AHB-APB Bridge B part. The prefix of AHB-APB Bridge B is 0x41xx-xxxx and there are entries with 0x40xx-xxxx which is complete wrong.